backend/sozisel/test/sozisel_web/schema/session_resources/session_resource_queries_test.exs

```diff
@@ -0,0 +1,222 @@
+    setup do
+      u1 = insert(:user)
+      u2 = insert(:user)
+      session = insert(:session)
+      participant = insert(:participant)
+      [conn: test_conn(u1), u1: u1, u2: u2, session: session, participant: participant]
+    end
+
+    test "get all files for presenter", ctx do
+      sr = insert(:session_resource, %{user_id: ctx.u1.id})
+      insert(:session_resource, %{user_id: ctx.u2.id})
+      insert(:session_resource, %{user_id: ctx.u2.id})
+
+      sr_filename = sr.filename
+      sr_id = sr.id
+
+      assert %{
+               data: %{
+                 "files" => [
+                   %{
+                     "filename" => ^sr_filename,
+                     "id" => ^sr_id
+                   }
+                 ]
+               }
+             } = run_query(ctx.conn, @get_files_query, %{})
+
+      assert %{
+               data: %{
+                 "files" => files
+               }
+             } = run_query(test_conn(ctx.u2), @get_files_query, %{})
+
+      assert length(files) == 2
+    end
+
+    test "get all session_resources for presenter in session", ctx do
+      sr1 = insert(:session_resource, %{user_id: ctx.u1.id})
+      sr2 = insert(:session_resource, %{user_id: ctx.u2.id})
+
+      insert(:session_resource_link, %{
+        session_resource_id: sr1.id,
+        session_id: ctx.session.id,
+        is_public: false
+      })
+
+      insert(:session_resource_link, %{
+        session_resource_id: sr2.id,
+        session_id: ctx.session.id,
+        is_public: true
+      })
+
+      assert %{
+               data: %{
+                 "sessionResourcesPresenter" => files
+               }
+             } =
+               run_query(ctx.conn, @get_session_resources_presenter_query, %{id: ctx.session.id})
+
+      assert length(files) == 2
+    end
+
+    test "get all session_resources for participant in session", ctx do
+      sr1 = insert(:session_resource, %{user_id: ctx.u1.id})
+      sr2 = insert(:session_resource, %{user_id: ctx.u2.id})
+
+      srl =
+        insert(:session_resource_link, %{
+          session_resource_id: sr1.id,
+          session_id: ctx.session.id,
+          is_public: true
+        })
+
+      insert(:session_resource_link, %{
+        session_resource_id: sr2.id,
+        session_id: ctx.session.id,
+        is_public: false
+      })
+
+      srl_filename = sr1.filename
+      srl_id = srl.id
+
+      assert %{
+               data: %{
+                 "sessionResourcesParticipant" => [
+                   %{
+                     "filename" => ^srl_filename,
+                     "id" => ^srl_id
+                   }
+                 ]
+               }
+             } =
+               run_query(ctx.conn, @get_session_resources_participant_query, %{
+                 id: ctx.session.id,
+                 token: ctx.participant.token
+               })
+    end
+
+    test "download file by presenter", ctx do
+      sr = insert(:session_resource, %{user_id: ctx.u1.id})
```

Comment:
please don't use those shortcuts of names, use full names `session_resource`, I mean in all those tests. It is a little bit unreadable and does not bring any benefits (unless you want to save space in github repo :P )